Bricks/net/http/test.cc

```diff
@@ -502,65 +502,65 @@ TEST(PosixHTTPServerTest, ChunkedBoundaryCases) {
       "resize the buffer 800 -> 1600\n",
       "process a 792 bytes long chunk\n",
       "memmove 2 bytes from offset 797 to the beginning\n",
-      "read 1 bytes (buffer offset 2)\n"
+      "read 1 bytes while requested 1597 (buffer offset 2)\n"
     },
     {// Test case 1.1 (99 bytes header).
-      "read 99 bytes (buffer offset 0)\n",
+      "read 99 bytes while requested 99 (buffer offset 0)\n",
       "resize the buffer 100 -> 200\n",
       "http header is parsed\n",
-      "read 199 bytes (buffer offset 0)\n",
+      "read 199 bytes while requested 199 (buffer offset 0)\n",
       "resize the buffer 200 -> 400\n",
       "process a 90 bytes long chunk\n",
-      "memmove 100 bytes from offset 99 to fit an entire chunk\n",
-      "read 299 more bytes of a chunk\n",
+      "memmove 100 bytes from offset 99 to fit the entire chunk\n",
+      "read 299 more bytes of a chunk at offset 100\n",
       "process a 399 bytes long chunk\n",
-      "read 399 bytes (buffer offset 0)\n",
+      "read 399 bytes while requested 399 (buffer offset 0)\n",
       "resize the buffer 400 -> 800\n",
       "process a 393 bytes long chunk\n",
       "memmove 1 bytes from offset 398 to the beginning\n",
-      "read 798 bytes (buffer offset 1)\n",
+      "read 798 bytes while requested 798 (buffer offset 1)\n",
       "resize the buffer 800 -> 1600\n",
       "process a 792 bytes long chunk\n",
       "memmove 2 bytes from offset 797 to the beginning\n",
-      "read 1 bytes (buffer offset 2)\n"
+      "read 1 bytes while requested 1597 (buffer offset 2)\n"
     },
     {// Test case 1.2 (98 bytes header).
-      "read 99 bytes (buffer offset 0)\n",
+      "read 99 bytes while requested 99 (buffer offset 0)\n",
       "resize the buffer 100 -> 200\n",
       "http header is parsed\n",
       "memmove 1 bytes from offset 98 to the beginning\n",
-      "read 198 bytes (buffer offset 1)\n",
+      "read 198 bytes while requested 198 (buffer offset 1)\n",
       "resize the buffer 200 -> 400\n",
       "process a 90 bytes long chunk\n",
-      "memmove 100 bytes from offset 99 to fit an entire chunk\n",
-      "read 299 more bytes of a chunk\n",
+      "memmove 100 bytes from offset 99 to fit the entire chunk\n",
+      "read 299 more bytes of a chunk at offset 100\n",
       "process a 399 bytes long chunk\n",
-      "read 399 bytes (buffer offset 0)\n",
+      "read 399 bytes while requested 399 (buffer offset 0)\n",
       "resize the buffer 400 -> 800\n",
       "process a 393 bytes long chunk\n",
       "memmove 1 bytes from offset 398 to the beginning\n",
-      "read 798 bytes (buffer offset 1)\n",
+      "read 798 bytes while requested 798 (buffer offset 1)\n",
       "resize the buffer 800 -> 1600\n",
       "process a 792 bytes long chunk\n",
       "memmove 2 bytes from offset 797 to the beginning\n",
-      "read 1 bytes (buffer offset 2)\n"
+      "read 1 bytes while requested 1597 (buffer offset 2)\n"
```

Comment:
But what's wrong with these offsets? Previous line in the journal shows, that unparsed 2 bytes was moved to the beginning of the buffer (it is `0\r`, which can't be parsed until the final `\n` is read). So the next portion of data is read at offset 2..